src/services/findAllReferences.ts

```diff
@@ -709,6 +709,16 @@ namespace ts.FindAllReferences.Core {
         return exposedByParent ? scope.getSourceFile() : scope;
     }
 
+    /** Used as a quick check for whether a symbol is used at all in a file (besides its definition). */
+    export function isSymbolReferencedInFile(definition: Identifier, checker: TypeChecker, sourceFile: SourceFile) {

```

Comment:
It probably wouldn't hurt to test this method.

Reply:
#22134